tests/LINEBot/HTTPClient/CurlHTTPClientTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace LINE\Tests\LINEBot\HTTPClient;
+
+use LINE\LINEBot\Constant\Meta;
+use LINE\LINEBot\HTTPClient\CurlHTTPClient;
+
+class CurlHTTPClientTest extends \PHPUnit_Framework_TestCase
+{
+    private static $reqMirrorPath = __DIR__ . '/../../../devtool/req_mirror';
+    private static $reqMirrorPort;
+    private static $reqMirrorPID;
+
+    public static function setUpBeforeClass()
+    {
+        if (file_exists(CurlHTTPClientTest::$reqMirrorPath)) {
+            if (empty(CurlHTTPClientTest::$reqMirrorPort)) {
+                $sock = socket_create(AF_INET, SOCK_STREAM, SOL_TCP);
+                socket_bind($sock, '127.0.0.1', 0);
+                socket_getsockname($sock, $address, CurlHTTPClientTest::$reqMirrorPort);
+                socket_close($sock);
+            }
+
+            if (empty(CurlHTTPClientTest::$reqMirrorPID)) {
+                $out = [];
+                $cmd = sprintf(
+                    'nohup %s --port %d > /dev/null & echo $!',
+                    CurlHTTPClientTest::$reqMirrorPath,
+                    CurlHTTPClientTest::$reqMirrorPort
+                );
+                exec($cmd, $out);
+                CurlHTTPClientTest::$reqMirrorPID = $out[0];
+                sleep(1); // XXX
+            }
+
+            return;
+        }
+
+        echo <<<"EOS"
+\n[WARN] req_mirror does not exist. Please try to execute `make install-devtool`
+[WARN] Skip CurlHTTPClientTest\n
```

Comment:
Does that mean `don't skip test, to fail instead`?
rel: https://github.com/line/line-bot-sdk-php/pull/38/files#diff-d0ccb18514d6f6cb88ff030b1b5697d0R69